extensions/wikia/ArticleAsJson/ArticleAsJson.class.php

```diff
@@ -65,7 +65,8 @@ private static function renderImage( $media, $id ) {
 					'title' => $media['title'],
 					'fileUrl' => $media['fileUrl'],
 					'caption' => $media['caption'] ?? '',
-					'link' => $media['link'] ?? '',
+					'href' => $media['href'],
+					'isLinkedByUser' => $media['isLinkedByUser'],
```

Comment:
I know it's minor but I'm not sure if the name is ok here. What do you think about simple `isLink`?